getLocationLambda.js

```diff
@@ -1,18 +1,10 @@
 const AWS = require('aws-sdk');
 
+const Ajv = require("ajv")
+const ajv = new Ajv()
+
 const dynamo = new AWS.DynamoDB.DocumentClient();
 
-/**
- * Demonstrates a simple HTTP endpoint using API Gateway. You have full
- * access to the request and response payload, including headers and
- * status code.
- *
- * To scan a DynamoDB table, make a GET request with the TableName as a
- * query string parameter. To put, update, or delete an item, make a POST,
- * PUT, or DELETE request respectively, passing in the payload to the
- * DynamoDB API as a JSON body.
- */
- 
  const getParkingData = async () => {
```

Comment:
TODO:

There must be a better way to do this. As we scale up, trying to pull the whole database at a time and filtering after the fact won't work.

Reply:
<img width="863" alt="image" src="https://user-images.githubusercontent.com/16601367/168934235-20442acb-7b09-40bd-8efe-0237c9c09f1e.png">